_studio/shared/src/mfx_vpp_vaapi.cpp

```diff
@@ -1650,14 +1653,12 @@ mfxStatus VAAPIVideoProcessing::Execute_Composition_TiledVideoWall(mfxExecutePar
             m_pipelineParam[i].blend_state = &blend_state[i];
         }
 
-#if defined(LINUX_TARGET_PLATFORM_BXT) || defined(LINUX_TARGET_PLATFORM_BXTMIN)
         m_pipelineParam[i].pipeline_flags |= VA_PROC_PIPELINE_SUBPICTURES;
-        m_pipelineParam[i].filter_flags   |= VA_FILTER_SCALING_HQ;
-#else
-        m_pipelineParam[i].pipeline_flags  |= VA_PROC_PIPELINE_FAST;
-#endif
+        m_pipelineParam[i].filter_flags |= VA_FILTER_SCALING_HQ;
+
         m_pipelineParam[i].filters      = 0;
         m_pipelineParam[i].num_filters  = 0;
+        m_pipelineParam[i].output_background_color &= 0x00FFFFFF;
```

Comment:
why `&=`? What you really try to program here? I have some feeling that idea is to program `m_pipelineParam[i].output_background_color=0`...